src/main/java/uk/gov/justice/hmpps/prison/service/BookingService.java

```diff
@@ -195,6 +208,9 @@ public BookingService(final BookingRepository bookingRepository,
         this.offenderSentenceRepository = offenderSentenceRepository;
         this.availablePrisonIepLevelRepository = availablePrisonIepLevelRepository;
         this.offenderRestrictionRepository = offenderRestrictionRepository;
+        this.healthProblemTypeReferenceCodeRepository = healthProblemTypeReferenceCodeRepository;
```

Comment:
Class getting very big - I suggest moving the health stuff to a new HealthService

Reply:
done